velox/substrait/VariantToVectorConverter.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/substrait/VariantToVectorConverter.h"
+#include "velox/vector/ComplexVector.h"
+#include "velox/vector/FlatVector.h"
+namespace facebook::velox::substrait {
+
+template <TypeKind KIND>
+VectorPtr setVectorFromVariantsByKind(
```

Comment:
put this into anonymous namespace

Reply:
I extracted it into a separate PR here. #2857